apps/site/components/Containers/NavBar/index.tsx

```diff
@@ -53,17 +53,21 @@ const NavBar: FC<NavbarProps> = ({
         <Link className={style.nodeIconWrapper} href="/" aria-label="Home">
           <WithNodejsLogo />
         </Link>
-
         <Label.Root
-          onClick={() => setIsMenuOpen(prev => !prev)}
           className={style.sidebarItemTogglerLabel}
           htmlFor="sidebarItemToggler"
         >
           {navInteractionIcons[isMenuOpen ? 'close' : 'show']}
         </Label.Root>
       </div>
 
-      <input className="peer hidden" id="sidebarItemToggler" type="checkbox" />
+      <input
+        className={`peer ${style.sidebarItemToggler}`}
+        id="sidebarItemToggler"
+        type="checkbox"
+        onChange={e => setIsMenuOpen(() => e.target.checked)}
+        aria-label={`${isMenuOpen ? 'Close' : 'Show'} navigation menu`}
```

Comment:
This should be an i18n translation text. Can you add on packages/i18n/locales/en.json?

Reply:
OK but I ve no idea how to use it

Reply:
```tsx
const t = useTranslation()

return (<span>{t("i18n.key.feaures")}</span>)
```

I hope this example is sufficient

Reply:
BTW i18n key use unicode message format